src/Main.java

```diff
@@ -18,14 +18,18 @@ public static void main(String[] args) {
 //Задача 2
         int clientDeviceYear=2015;
         int clientOsOne=1;
-        if (clientDeviceYear<=2015 && clientOsOne>=1) {
+        if (clientDeviceYear<2015 && clientOsOne>=1) {
             System.out.println("Установите облегченную версию приложения для iOS по ссылке");
-        } else {
+        } else if (clientDeviceYear<2015 && clientOsOne>=0) {
             System.out.println("Установите облегчëнную версию приложения для Android по ссылке");
+        } else if (clientDeviceYear>=2015 && clientOsOne>=1) {
```

Comment:
```suggestion
        } else if (clientDeviceYear>=2015 && clientOsOne==1) {
```
clientOsOne>=1 ? должно быть равно